### ISSUE TITLE
using the pretrain model or not? //使用开源模型参数作为预训练模型是否可以？

### ISSUE THREAD

**Issue by PanffeeReal** [WongKinYiu/yolov9#643]
Dear authors:
    Thanks for opensource the wonderful code.
    I'd like to know if we should use the open access pretrained models, when doing comparison experiments. Due to concerns about overfitting, are the default parameter values ​​in the training script, such as learning rate, only applicable to training from scratch, or are they still applicable to fine-tuning based on the released model?

Thanks ~

*********************************************************
尊敬的作者：
感谢您开源这些精彩的代码。
我想知道在进行对比实验时，我们是否应该使用开放获取的模型参数作为预训练模型。
由于担心过拟合的问题，且训练脚本中的参数默认值，例如学习率是只适用于从头训练，或是依然适用于在发布的模型基础上微调？
希望得到您的解答，感谢~
